packit_service/config.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         bugzilla_url: str = "",
         bugzilla_api_key: str = "",
         pr_accepted_labels: List[str] = None,
+        gitlab_webhook_token: str = "",
```

Comment:
I would pick a list of allowed token's if it does not complicate anything.

Reply:
Sure.

Reply:
Done made it into the list!

Reply:
Thanks!